src/applications/disability-benefits/all-claims/pages/incomeDetails.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import { unemployabilityTitle } from '../content/unemployabilityFormIntro';
+
+import { isValidYear } from '../validations';
+
+const incomeDescription = (
+  <div>
+    <h3>Income details</h3>
+    <p>
+      Now we‘re going to ask you about your income history. Please provide your
+      gross income, which is all the money you earned through employment for the
+      year before taxes. You should include military pay but you don‘t need to
+      include Social Security benefits, VA benefits, stock dividends, or
+      investment income. If you can‘t remember the exact dollar amount, you can
+      give an estimated amount.
+    </p>
+  </div>
+);
+
```

Comment:
Should the content be moved to `content`?

Reply:
Yeah content updated!